### ISSUE TITLE
Question about the S3DIS Validation Set

### ISSUE THREAD

**Issue by JNZZZ** [tsunghan-mama/ReDAL#12]
Thank you for your great work!
In your code I notice that you divide the S3DIS dataset into train set and val set(area 5, like other projects have done). But when a new active learning iteration begins , you choose the checkpoint that performs the best miou in val set to initialize parameters this iteration. I'm wondering if this may causes information leakage and makes the model overfitting.
